developer_setup.md

```diff
@@ -176,6 +176,37 @@
   launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.memcached.plist
   ```
 
+##### Problems
```

Comment:
There is already a Troubleshooting section at the bottom...can you please move this down there?